content/tutorial-build-run-rippled-ubuntu.md

```diff
@@ -55,41 +55,53 @@ These instructions use Ubuntu's APT (Advanced Packaging Tool) to install the sof
 
 8. Install `wget` to be able to download Boost in the next step.
 
-        sudo apt-get install wget
+        sudo apt-get -y install wget
 
-8. Compile Boost.
+9. Compile Boost.
 
-    Starting in `rippled` 0.90, the compatible Boost version is 1.65.0. Because Boost version 1.65.0 isn't available in the Ubuntu 16.04 repos, you'll need to compile it yourself. ***TODO: Per Scott D, the recommendation will be either 1.65.0 or 1.66.0+ for 0.90.0. Per Scott D, test with 1.65.0 first; this doc reflects that test. If this PR is accepted, the recommendation will be 1.66.0+***
+    Starting in `rippled` 0.90, the compatible Boost version is 1.66.0. Because Boost version 1.66.0 isn't available in the Ubuntu 16.04 repos, you'll need to compile it yourself.
```

Comment:
Nitpick: "you'll need to"→"you must" (avoids future tense, simpler for non-native English speakers)